tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -120,19 +122,84 @@ static void to_display_format(const uint8_t *src, size_t size, char *dst)
 	}
 }
 
+#if DT_NODE_HAS_PROP(DT_PATH(zephyr_user), cs_loopback_gpios)
+
+static const struct gpio_dt_spec cs_loopback_gpio =
+			GPIO_DT_SPEC_GET_OR(DT_PATH(zephyr_user), cs_loopback_gpios, {0});
+static struct gpio_callback cs_cb_data;
+static K_SEM_DEFINE(cs_sem, 0, UINT_MAX);
```

Comment:
a simple integer would just do here.

Reply:
Maybe, and I thought about it, but I didn't want to make any assumptions about the different platforms